src/lib/viewport-uniforms.js

```diff
@@ -155,6 +155,8 @@ export function getUniformsFromViewport(viewport, {
     modelViewMatrix: new Float32Array(modelViewMatrix),
 
     // Screen size
+    // _Q2# : Verify and remove multiplicaiton with devicePixelRatio (assuming
```

Comment:
Yes. `DeckGL` defined `viewport` from user provided props. The uniform is used by layers to project pixel size to clipspace, which requires the actual canvas size.